packages/landingpage-new/translations/locales/de/home.json

```diff
@@ -59,49 +59,48 @@
+      },
+      {
+        "title": "Warum verwendet nicht jeder Web-Components?",
+        "content": "Web-Components sind ein relativ neuer Standard, welche erst seit 2018 von den gängigsten Browsern unterstützt wird. Außerdem ist die Pflege einer solchen Bibliothek initial aufwändiger, da jedes Framework (React, Angular, ...) seine eigenen Besonderheiten mit sich bringt. Ist dieser Hürde allerdings genommen, kann sich immenser Wartungsaufwand gespart werden, da neue Komponenten eimalig entwickelt werden müssen und danach im jeden Framework zur Verfügung steht."
+      },
+      {
+        "title": "Sind die inovex Elemente Open Source?",
         "content": "Ja! Die Inovex Elements-Bibliothek ist Open Source und unter der MIT-Lizenz kostenlos verfügbar."
       },
       {
-        "id": 2,
         "title": "Wie kann ich die Elemente verwenden?",
         "content": "Sie können die Inovex Elements Web Components-Bibliothek in jedem Webprojekt verwenden, da sie auf Webkomponenten basieren, einem Framework-unabhängigen Standard. Um die Sache noch einfacher zu machen, bieten wir verschiedene Pakete für die gängigen Frameworks wie Angular, React und Vue an. Schauen Sie sich unsere Integrationsanleitungen an, um zu sehen, wie Sie sie im Detail verwenden können."
       },
       {
-        "id": 3,
+        "title": "Welche Anforderungen haben die inovex Elements?",
+        "content": "Es gibt keine Anforderungen, außer dass Ihre Anwendung eine Webanwendung sein muss. Die Inovex Elements-Bibliothek unterstützt die neueste Version aller gängigen Browser. Beachten Sie, dass unsere Komponenten derzeit für umfassende Desktop-Anwendungen optimiert sind."
+      },
+      {
         "title": "Wo ist der beste Ort, um anzufangen, wenn ich neu bin?",
         "content": "Schauen Sie sich unsere Getting Started-Seite an, die die Grundlagen erklärt und Sie durch Ihre ersten Schritte mit den Inovex Elements führt. Es bietet auch detaillierte Anweisungen, wie Sie die Inovex Elements in Ihre eigenen Projekte einbinden können."
       },
       {
-        "id": 4,
-        "title": "Gibt es Beispiele, in denen die Inovex-Elemente verwendet wurden?",
-        "content": "Die Inovex-Elemente werden hauptsächlich für unsere internen Web-Tools bei Inovex GmbH verwendet. Um ein Gefühl dafür zu bekommen, wie die Elemente aussehen und sich anfühlen, können Sie sich unsere Beispielprojekte ansehen."
+        "title": "Gibt es Beispiele, in denen die inovex Elements verwendet wurden?",
+        "content": "Die inovex Elements werden hauptsächlich für unsere internen Web-Tools bei Inovex GmbH verwendet. Um ein Gefühl dafür zu bekommen, wie die Elemente aussehen und sich anfühlen, können Sie sich unsere Beispielprojekte ansehen."
```

Comment:
I think we should write inovex instead of Inovex

Reply:
If yes, please check all other occurrences

Reply:
Yes, that's why I changed it. It should be inovex everywhere